test_caseinsensitivedict.py

```diff
@@ -100,12 +108,27 @@ def test_keys(self):
         # popitem()
         #     Remove and return a (key, value) pair from the dictionary.  Pairs
         #     are returned in LIFO order.
+    def test_popitem(self):
+        myinstance = CaseInsensitiveDict()
+        myinstance['Kaisa'] = 5
+        self.assertIn('Kaisa', myinstance)
+        item = myinstance.popitem()
+        self.assertEqual(item, ('Kaisa', 5))
+        self.assertNotIn(item, myinstance)
```

Comment:
what kinds of things can we use the 'in' operator on a dictionary with? what kind of thing are you using here?

Reply:
can we try adding 3 items and then popping all 3 (testing each value as you go), and confirming it is empty at the end?